portfolio/src/main/webapp/script.js

```diff
@@ -16,11 +16,39 @@
  * Adds a random greeting to the page.
  */ 
 
-function changeColor() { 
-    let x = Math.floor(Math.random() * 256);
-    let y = Math.floor(Math.random() * 256);
-    let z = Math.floor(Math.random() * 256);
-    let rgb = "rgb(" + x + "," + y + "," + z + ")";
-    document.getElementById('box').style.backgroundColor = rgb;
-    document.getElementById('color').innerHTML = rgb;
+// function changeColor() { 
+//     let x = Math.floor(Math.random() * 256);
+//     let y = Math.floor(Math.random() * 256);
+//     let z = Math.floor(Math.random() * 256);
+//     let rgb = "rgb(" + x + "," + y + "," + z + ")";
+//     document.getElementById('box').style.backgroundColor = rgb;
+//     document.getElementById('color').innerHTML = rgb;
+// }
+
+// @Webservlet tells what '/' to put in fetch look at DataServlet.java
+
+// async function getFetchRequest() {
+//     const response = await fetch('/data');
+//     const text = await response.text();
+//     document.getElementById('fetch-text').innerText = text;
+// }
+
+function getArrayMessages() {
+    fetch('/data').then(response => response.json()).then((messages) => {
+    console.log(messages);
+    const textId = document.getElementById('fetch-text');
+    textId.innerHTML = '';
+    textId.appendChild(
+        createListElement('text1: ' + messages.text1));
+    textId.appendChild(
+        createListElement('text2: ' + messages.text2));
+    textId.appendChild(
+        createListElement('text3: ' + messages.text3));
+    });
 }
+
+function createListElement(message) {
+        const liElement = document.createElement('li');
+        liElement.innerText = message;
+        return liElement;
```

Comment:
nit: The indentation of the code inside of this method here appears to be inconsistent with the indentation of the rest of the code (appears to be 8 instead of 4 from a glance).